Src/FluentAssertions/Primitives/BooleanAssertions.cs

```diff
@@ -118,6 +118,16 @@ public AndConstraint<TAssertions> NotBe(bool unexpected, string because = "", pa
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+    public AndConstraint<TAssertions> Imply(bool expectation, string because = "", params object[] becauseArgs)
```

Comment:
Don't we use `expected` most places?

Reply:
True

Reply:
But: this is not the expectation.. this is the second parameter for the implication.. so "operand" would fit better IMO

Reply:
I have called it "implicator" by now.. Feel free to disagree :D